api-reference/10 UI Widgets/dxFileManager/1 Configuration/currentPathKeys.md

```diff
@@ -5,7 +5,6 @@ default: []
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies an array of path keys to the current location.
```

Comment:
Is "path key" a generally understood term? An example would be helpful.

Reply:
Yes, it is a common term for file management.